src/retasc/product_pages_api.py

```diff
@@ -14,66 +15,32 @@ class ProductPagesApi:
     """
 
     def __init__(self, api_url: str):
-        self.api_url = api_url
+        self.api_url = api_url.rstrip("/")
         self.session = requests_session.requests_session()
 
-    def api_url_releases(self, release_name: str | None = None) -> str:
-        return f"{self.api_url.rstrip('/')}/releases/{release_name or ''}"
-
-    def api_url_release_schedule_tasks(self, release_name: str) -> str:
-        return f"{self.api_url_releases(release_name)}/schedule-tasks"
-
     @cache
-    def active_releases(
-        self,
-        product_shortname: str,
-        fields: str = "shortname,product_shortname,ga_date,phase_display",
-    ) -> list[dict]:
-        """
-        https://{pp_release_url}?product__shortname={product_shortname}&phase__lt=Unsupported&fields=a,b,c
-        The response have the following fields:shortname,product_shortname,ga_date,phase_display
-        phase__lt=Unsupported means "any supported phase" (these are less than Unsupported).
-        """
-
+    def active_releases(self, product_shortname: str) -> list[str]:
+        """Gets list of active release names."""
         opt = {
             "product__shortname": product_shortname,
             "phase__lt": "Unsupported",
-            "fields": fields,
+            "fields": "shortname",
```

Comment:
`ga_date,phase_display` think we will need those two fields in future, but can be added when it needs

Reply:
Why would we need those? I think "GA" task with the date is always part of the schedule. Not sure how would we use `phase_display`.

Reply:
When create the spreadsheet, but they might need or not need in future. Remove them are fine now.
My concern is if you need to add some fields, you need to update the code, which makes it not very flexible.

Reply:
Let's keep it simpler for now. I was thinking about passing the dict with schedules to the templates so you would be able to refer to them (something like `{{ schedules["GA"].start_date }}`).